exec/totemsrp.c

```diff
@@ -2619,6 +2619,7 @@ static int orf_token_remcast (
 
 	sort_queue_item = ptr;
 
+	instance->stats.orf_token_tx++;
```

Comment:
Are you sure about this? This looks suspicious because mcast send is then called, token is unicast.

Reply:
TBH no I'm not 100% sure about it. The function is (re-)mcasting the ORF token so it seemed reasonable to include that's all. If you think it's wrong then I'll take it out.

Reply:
On further investigation I thing you're right, so I've removed that line. it doesn't affect the 'sanity' test results

Reply:
Thanks